Lab1/Calculater/Calculater/MyCalculater.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Numerics;
+
+namespace Calculater
+{
+    public class MyCalculater
+    {
+        public MyCalculater()
```

Comment:
https://coderoad.ru/2963742/%D0%9F%D1%83%D1%81%D1%82%D0%BE%D0%B9-%D0%BA%D0%BE%D0%BD%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D0%BE%D1%80-%D0%B8%D0%BB%D0%B8-%D0%B1%D0%B5%D0%B7-%D0%BA%D0%BE%D0%BD%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D0%BE%D1%80%D0%B0